UI/MapUI.gd

```diff
@@ -70,3 +85,14 @@ func _ready():
 		# When we finish the array for a floor we need to go to the next floor up,
 		# so increase the Y position to the new area the room will be displayed on.
 		position_for_next_room.y -= new_room_size.y + _padding_offset
+	
+	# Calculate the position of where we should put the rooms. 
+	# This is to account for in case the size of the rooms is smaller than the container we put it in
```

Comment:
isn't the size of the room always smaller than the container we put it in though ?

Reply:
Not necessarily! The width that we set for the container accounts for the padding as well, so if it exceeds the minimum width we're setting the container to the width that we calculate on Line 32.

For this screenshot, the size of the rooms is less than the size of the container since we have to set that `_MINIMUM_ROOM_WIDTH`
![image](https://github.com/Saplings-Projects/1M_sub/assets/5923102/2f771882-953f-492c-8c4f-e795d594b9ad)

For this screenshot, this size of the room exceeds the `MINIMUM_ROOM_WIDTH` hence we just keep the size of the rooms and just apply it to the container:
![image](https://github.com/Saplings-Projects/1M_sub/assets/5923102/4840473f-1c76-4ac8-b028-05f29992bb76)

Reply:
ah I see now !